{
    "/home/zhang/Packages/httpx/httpx0.12.1/_utils.py": [
        {
            "httpx._utils.Logger.trace": 221
        },
        {
            "httpx._utils.trace": 251
        },
        {
            "httpx._utils.as_network_error": 362
        }
    ],
    "/home/zhang/Packages/httpx/httpx0.12.1/_models.py": [
        {
            "httpx._models.URL.copy_with": 196
        },
        {
            "httpx._models.QueryParams.__init__": 294
        },
        {
            "httpx._models.Response.json": 848
        }
    ],
    "/home/zhang/Packages/httpx/httpx0.12.1/_exceptions.py": [
        {
            "httpx._exceptions.HTTPError.__init__": 12
        }
    ]
}